src/piskel-boot.js

```diff
@@ -18,6 +22,11 @@
     delete window.pskl_exports;
     delete window.loadDebugScripts;
     delete window.done;
+
+    // Run Piskel ready callbacks
+    for (var i in window.onPiskelReady) {
```

Comment:
Do not use for ... in to loop on the array. either a regular for loop or a forEach.